cranelift/assembler-x64/src/main.rs

```diff
@@ -1,7 +1,9 @@
-//! Print the path to the generated code.
+//! Print the contents of the generated `assembler_definitions.isle`.
 
 fn main() {
-    for path in cranelift_assembler_x64::generated_files() {
-        println!("{}", path.display());
-    }
+    println!(
+        "{}",
+        std::str::from_utf8(&cranelift_assembler_x64::assembler_definitions_isle_contents())
+            .unwrap()
+    );
```

Comment:
I think it would be nice if this executable kept showing a path to a file on the disk. It is for debugging only and emitting the entire file to stdout would be annoying. Maybe it could write the file to somewhere in the directory from which it is invoked?